test/model/content_test.dart

```diff
@@ -528,6 +527,20 @@ class ContentExample {
       ]),
     ]));
 
+  // A test message to test the fallback behaviour of KaTeX implementation.
+  static final mathInlineUnknown = ContentExample.inline(
+    'inline math',
+    r"$$ \lambda $$",
```

Comment:
Since this HTML isn't something the server actually generates, this `markdown` field should be null. See its doc:
```dart
  /// The Zulip Markdown source, if any, that the server renders as [html].
  ///
  /// This is useful for reproducing the example content for live use in the
  /// app, and as a starting point for variations on it.
  ///
  /// Currently the test suite does not verify the relationship between
  /// [markdown] and [html].
  ///
  /// If there is no known Markdown that a Zulip server can render as [html],
  /// then this should be null and a comment should explain why the test uses
  /// such an example.
  final String? markdown;
```

So e.g., following the example `emojiUnicodeClassesFlipped`:

```suggestion
  static final mathInlineUnknown = ContentExample.inline(
    'inline math',
    null, // r"$$ \lambda $$" (hypothetical server variation)
```